node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/component/Environment.java

```diff
@@ -62,14 +64,16 @@ public Environment(ConfigServerConfig configServerConfig) {
              getEnvironmentVariable(ENVIRONMENT),
              getEnvironmentVariable(REGION),
              new PathResolver(),
-             Optional.of(getEnvironmentVariable(COREDUMP_FEED_ENDPOINT)));
+             Optional.of(getEnvironmentVariable(COREDUMP_FEED_ENDPOINT)),
+             NodeType.host);
     }
 
     public Environment(ConfigServerConfig configServerConfig,
                        String hostedEnvironment,
                        String hostedRegion,
                        PathResolver pathResolver,
-                       Optional<String> coreDumpFeedEndpoint) {
+                       Optional<String> coreDumpFeedEndpoint,
+                       NodeType nodeType) {
```

Comment:
You must then also change hosted.

In hosted there's a HostType. We should instead use NodeType.